client/my-sites/themes/theme-showcase.scss

```diff
@@ -102,6 +102,112 @@ body.is-section-themes-i4 {
 	}
 }
 
+body.is-section-themes-i4-2 {
+	.themes__selection .themes-list .theme {
+		background: none;
+		border-radius: 2px;
+		box-shadow: none;
+		margin: 0 16px 48px;
+		overflow: hidden;
+
+		&.is-active {
+			.theme__info {
+				align-items: center;
+				flex-direction: row;
+				gap: 0;
+				padding: 0 24px;
+			}
+		}
+
+		.theme__content {
+			padding-bottom: 76px;
+		}
+
+		.theme__img {
+			box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.05);
+		}
+
+		.theme__info {
+			align-items: flex-start;
+			box-sizing: border-box;
+			display: flex;
+			flex-direction: row;
+			flex-wrap: wrap;
+			height: 75px;
+			gap: 4px;
+			padding-top: 24px;
+		}
+
+		.theme__info-title {
+			font-size: 1rem;
+			line-height: 24px;
+			margin: 0;
+			padding: 0;
+
+			.theme:not(.is-active) & {
```

Comment:
This doesn't work. It will become the first one as followed

```css
.theme:not(.is-active) body.is-section-themes-i4-2 .themes__selection .themes-list .theme .theme__info-title {
    color: var(--color-neutral-100)
}
```

Reply:
Ah good catch, thank you!